feat: Add all Enterprise MFEs to devstack.

pwnage101 opened the pull request:
Added 3 MFEs:

- frontend-app-admin-portal
- frontend-app-learner-portal-enterprise
- frontend-app-enterprise-checkout

Also fix bug where enterprise-subsidy was missing from a list, preventing `make enterprise-subsidy-logs` -esque commands from working.

robrap commented:
@macdiesel @staubina: Looks like I got my headings backwards.  I'm going to clean that up and squash in a minute.

robrap commented:
@macdiesel @staubina: Headers are fixed.  This is ready for review.

pwnage101 committed:
feat: Add all Enterprise MFEs to devstack.

Added 3 MFEs:

- frontend-app-admin-portal
- frontend-app-learner-portal-enterprise
- frontend-app-enterprise-checkout

Also fix bug where enterprise-subsidy was missing from a list,
preventing `make enterprise-subsidy-logs` type commands from working.

pwnage101 commented on docs/service_list.rst:
```diff
@@ -10,73 +10,79 @@ Most developers will be best served by working with specific combinations of the
++-------------------------------------------+-------------------------------------+----------------+--------------+
+| `enterprise-subsidy`_                     | http://localhost:18280              | Python/Django  | Extra        |
++-------------------------------------------+-------------------------------------+----------------+--------------+
+| `frontend-app-admin-portal`_              | http://localhost:1991               | MFE (React.js) | Extra        |
++-------------------------------------------+-------------------------------------+----------------+--------------+
+| `frontend-app-learner-portal-enterprise`_ | http://localhost:8734               | MFE (React.js) | Extra        |
++-------------------------------------------+-------------------------------------+----------------+--------------+
+| `frontend-app-enterprise-checkout`_       | http://localhost:1989               | MFE (React.js) | Extra        |
```
Only added these 3 rows.
